WordPress/src/main/java/org/wordpress/android/ui/WPBottomSheetDialogFragment.kt

```diff
@@ -0,0 +1,5 @@
+package org.wordpress.android.ui
+
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+
+open class WPBottomSheetDialogFragment : BottomSheetDialogFragment()
```

Comment:
Why have you added this class?

Reply:
Currently it does nothing, but the original logic behind it was to have a place where to host common parts like the management of the bottom sheet reduced width for tablets or expanded state in landscape etc...these things are still under design definition but we will probably need to manage. Based on this reasoning I would keep it but open to change my mind of course. Wdyt?

Reply:
I think I'd rather add it once we need it. Maybe it would be possible to not use composition and not inheritance once we actually start working on these features and this parent class will kind of nudge us to use it :-). But I don't have a strong opinion about it :-)

Reply:
Let's remove it for now until we will work on it 😊; probably composition/delegation sounds better in any case. Changed here 81498eb89be0208efdc9e889e05db92df6bef97e